client/js/utils/ajax.js

```diff
@@ -2,10 +2,12 @@ import request from 'request';
 import when from 'when';
 
 
+// FIXME: replace with XMLHTTPRequest
```

Comment:
The index of parks is already over 50KB. Several of the various data files are several times that. Also, as I mentioned, anyone who has been on the Internet in the last decade will have jQuery cached if we're getting it from a big cdn. For reference, what's currently deployed is already 0.7MB. I assume that's mostly React.

It seems like wrapping the few places we'll need to make api calls in `prfun`'s `Promise.resolve` would solve everything, and that's only assuming the auto module wrapping thing doesn't work on jQuery.